lib/webgl.js

```diff
@@ -26,8 +26,8 @@ function createCanvas (element, onDone, pixelRatio) {
     var h = window.innerHeight
     if (element !== document.body) {
       var bounds = element.getBoundingClientRect()
-      w = bounds.right - bounds.left
-      h = bounds.bottom - bounds.top
+      w = bounds.width
+      h = bounds.height
```

Comment:
These are equivalent, right? As far as I can tell, they include padding, making them equal to `right - left`. Any difference regarding border? Just want to be sure we don't make drive-by subtle changes which might throw off people's pixel-perfect sites.

> Notice how the width/height are equal to the equal to its width/height + padding.
>
> Also note how the values of x/left, y/top, right, and bottom are equal to the absolute distance from the relevant edge of the viewport to that side of the element, in each case.

https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect

Reply:
I would be inclined to say these are equal, but let's not change this, there's no real benefit to doing this.